### DISCUSSION TITLE
posts/%E6%AC%A2%E8%BF%8E%E6%9D%A5%E5%88%B0%E6%88%91%E7%9A%84%E5%8D%9A%E5%AE%A2/

### DISCUSSION THREAD

**Discussion by giscus** [LingXi9374/LingXi9374.github.io#8] [General]
# posts/%E6%AC%A2%E8%BF%8E%E6%9D%A5%E5%88%B0%E6%88%91%E7%9A%84%E5%8D%9A%E5%AE%A2/

这是这个博客的第一篇文章

https://lingxi9374.github.io/posts/%E6%AC%A2%E8%BF%8E%E6%9D%A5%E5%88%B0%E6%88%91%E7%9A%84%E5%8D%9A%E5%AE%A2/

**Comment by LingXi9374** [comment]
评论测试

**Reply by LingXi9374** [reply]
回复测试

**Comment by jiangyin14** [comment]
### 牛逼

**Reply by jiangyin14** [reply]
### 牛逼
## 牛逼
# 太牛逼了

**Comment by LingXi9374** [comment]
代码测试

```Python
print("Hello World!")
```

**Comment by SeanDictionary** [comment]
哥们你可太强啦🥳🥳🥳

**Comment by YuYuBM** [comment]
我把你的网站加入我的书签了，可以方便找到

**Comment by YuYuBM** [comment]
只有我一个人知道吗？

**Comment by YuYuBM** [comment]
我的oid是22636

**Comment by YuYuBM** [comment]
你也是失传媒体爱好者吗？
